src/content/quickstart/vsphere_aws/_index.md

```diff
@@ -15,38 +15,56 @@ Please ensure that the tools you downloaded above are compatible with your OpenS
 please refer to the official [OpenShift documentation](https://docs.openshift.com/container-platform/).
 {{% /notice %}}
 
-{{< include "quickstart/openshift/setup_aws.md" >}}
+### Create and Deploy cluster-a on vSphere (On-Prem)
 
-### Deploy Cluster on vSphere (On-Prem)
-
-Create the necessary infrastructure on vSphere and ensure that your machines have direct internet access before starting the installation.
-See the [docs for deploying OCP 4.4 on
-vSphere](https://docs.openshift.com/container-platform/4.4/installing/installing_vsphere/installing-vsphere.html).
-
-Assuming that you deployed the cluster (say, cluster-a) with default network configuration, the Pod and Service CIDRs would be
+In this step you will deploy **cluster-a** using the default IP CIDR ranges:
 
 | Pod CIDR     | Service CIDR |
 |--------------|--------------|
 |10.128.0.0/14 |172.30.0.0/16 |
 
-Submariner creates a VXLAN overlay network in the local cluster and uses port 4800/UDP to encapsulate traffic from the worker nodes to the
-gateway nodes to preserve the source IP of the inter-cluster traffic.
-Ensure that firewall configuration on vSphere cluster allows 4800/UDP across all the worker nodes.
+Before you deploy an OpenShift Container Platform cluster that uses user-provisioned infrastructure, you must create the underlying
+infrastructure. Follow the [OpenShift documenation](https://docs.openshift.com/container-platform/) for installation instructions on
+supported versions of vSphere.
+
+Submariner gateway nodes need to be able to accept IPsec traffic. For on-premises clusters behind corporate firewalls, the default IPsec UDP
+ports might be blocked. To overcome this, Submariner supports NAT Traversal (NAT-T) with the option to set custom non-standard ports.
+In this example, we use UDP 4501 and UDP 501. Ensure that those ports are allowed on the gateway node.
 
-|  Protocol  |  Port  |     Description                              |
+Submariner also uses VXLAN to encapsulate traffic from the worker nodes to the gateway nodes. Ensure that firewall configuration on the
+vSphere cluster allows UDP 4800 across all the worker nodes.
+
+| Protocol   | Port   | Description                                  |
 |------------|--------|----------------------------------------------|
-|   UDP      |  4800  | overlay network for inter-cluster traffic    |
+| UDP        | 4800   | Overlay network for inter-cluster traffic    |
+| UDP        | 4501   | IPsec traffic                                |
+| UDP        | 501    | IPsec traffic                                |
 
-{{% notice tip %}}
+When the cluster deployment completes, directions for accessing your cluster, including a link to its web console and credentials for the
+`kubeadmin` user, display in your terminal.
 
-Although we are using the default OCP network configuration on vSphere, you can [install vSphere with a custom network
-configuration](https://red.ht/2WFjEVg).
+### Create and Deploy cluster-b on AWS
```

Comment:
@nyechiel not related to this patch, but there is a small typo in this page - https://github.com/submariner-io/submariner-website/blob/master/src/content/quickstart/openshift/run_prep_for_subm.md

Can you please fix the typo in cluser-b, thanks.

Reply:
Oh, good catch!